TheMovieDb/Controller/MovieInfoViewController.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  MovieInfoViewController.swift
+//  TheMovieDb
+//
+//  Created by Karla Rubiano on 26/10/21.
+//
+
+import UIKit
+import Kingfisher
+
+class MovieInfoViewController: UIViewController {
+    
+    var movie: Movie?
+    var movieID: Int?
+    var similarMoviesNames: String?
+    var recommendedMoviesNames: String?
+    var castMovie: String?
+    
+    @IBOutlet weak var titleMovie: UILabel!
+    
+    @IBOutlet weak var imageMovie: UIImageView!
+    
+    @IBOutlet weak var textOverview: UITextView!
+    
+    @IBOutlet weak var idMovie: UILabel!
+    
+    @IBOutlet weak var originalTitleMovie: UILabel!
+    
+    @IBOutlet weak var originalLanguageMovie: UILabel!
+    
+    @IBOutlet weak var popularityMovie: UILabel!
+    
+    @IBOutlet weak var adultMovie: UILabel!
+    
+    @IBOutlet weak var releaseDateMovie: UILabel!
+    
+    @IBOutlet weak var castInfoLabel: UILabel!
+    
+    @IBOutlet weak var similarMoviesLabel: UILabel!
+    
+    @IBOutlet weak var recommendationsLabel: UILabel!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        navigationController?.navigationBar.prefersLargeTitles = false
```

Comment:
You can configure the nav in a function ( It is like configuring the UI ). Try to keep cleaner the viewDidLoadMethod.